data/snippets.ts

```diff
@@ -545,12 +545,31 @@ const coding = [
     type: "template",
   },
   {
-    id: nanoid(),
     name: "Get Name of Current Branch",
+    id: nanoid(),
     text: "git branch --show-current",
     keyword: "git-current-branch",
     type: "template",
   },
+  {
+    name: "AppleScript Dialog",
+    id: nanoid(),
+    text: `try
+  set theResponse to (display dialog "{cursor}" default answer "Placeholder input" buttons {"Option 1", "Option 2", "Cancel"} default button 1 cancel button 3 with title "My Dialog" with icon note giving up after 1800 without hidden answer)
+  return theResponse
+on error
+  return "Canceled!"
+end try`,
+    keyword: "as-diag",
+    type: "template",
+  },
+  {
+    name: "AppleScript Choose From List",
+    id: nanoid(),
+    text: `set theResponse to (choose from list {"Option 1", "Option 2", "Option 3"} with title "Title" with prompt "{cursor}" default items {"Option 2"} OK button name "Continue" cancel button name "Cancel" without multiple selections allowed and empty selection allowed)
+return theResponse`,
+    keyword: "as-chls",
```

Comment:
missing `type: template`
```suggestion
    keyword: "as-chls",
    type: "template",    
```